compiler/src/iree/compiler/Codegen/LLVMGPU/TransformExtensions/LLVMGPUExtensions.cpp

```diff
@@ -1496,5 +1496,54 @@ void transform_dialect::PackSharedMemoryAllocOp::getEffects(
   transform::modifiesPayload(effects);
 }
 
+//===----------------------------------------------------------------------===//
+// CreateMatmulMfmaTileSizesOp
+//===----------------------------------------------------------------------===//
+
+DiagnosedSilenceableFailure
+transform_dialect::CreateMatmulMfmaTileSizesOp::apply(
+    transform::TransformRewriter &rewriter,
+    transform::TransformResults &results,
+    transform::TransformState &state) {
+  Builder builder(getTarget().getContext());
+  auto payload = state.getPayloadOps(getTarget());
+  Operation *target = *payload.begin();
+  ArrayRef<int64_t> shape =
+      llvm::cast<ShapedType>(target->getResult(0).getType()).getShape();
+  if (shape.size() != 2) {
+    return emitDefiniteFailure() << "matmul shape size should be 2";
+  }
+  SmallVector<int64_t> sz0, sz1;
+  SmallVector<Attribute> paramsArray0, paramsArray1;
+
+  // TODO: cover all shape cases
+  // (sz0[0] / sz1[0]) * (sz0[1] / sz1[1]) = 8
+  if (shape == ArrayRef<int64_t>({8192, 320})) {
+    sz0 = {512, 64}, sz1 = {64, 64};
```

Comment:
We usually split in two statements in MLIR/IREE codebase. I think it is a hidden convention, so I can't provide a style guide about it.